requirements.yml

```diff
@@ -4,6 +4,6 @@
   name: o2-priority.java
 
 - src: git@github.com:o2-priority/ansible-docker.git
-  version: master
+  version: jenkins-upgrade
```

Comment:
does this need reverting to `master`?

Reply:
also the o2-docker branch in `requirements.yml` is still pointing to `jenkins-upgrade`